includes/admin/class-wc-rest-print-receipt-controller.php

```diff
@@ -0,0 +1,165 @@
+<?php
+/**
+ * Class WC_REST_Print_Receipt_Controller
```

Comment:
As this class is specific to in-person payments, should we name it as such?

* Name is something like `WC_REST_IPP_Print_Receipt_Controller`
* Add this route in the recently added `WC_REST_Payments_Reader_Controller` since these receipts are intimately tied with card readers.

The second suggestion is probably a bit controversial, so feel free to ignore 😉

Reply:
Thanks for checking it @anu-rock 

I am happy to rename it. Please note that this controller is temporary, and only for testing purposes, I'd rather remove it, since we will work on the API on this issue IPP Styled Receipts: Print receipt (API) #3021.

Talking about the right place for it, there is also the option of `WC_REST_Payments_Charges_Controller` in `includes/admin/class-wc-rest-payments-charges-controller.php`. Maybe the mobile could call `payments/charges/<charge_id>/receipt`? But this is maybe to be decided when we implement the API?

Reply:
Finally I decided to add the route to `WC_REST_Payments_Reader_Controller` for now. We can consider other options when we implement the API.